index.html

```diff
@@ -60,24 +60,24 @@ <h2>Adventure Awaits</h2>
     <section class="content-destination">
       <h2>Pick Your Destination</h2>
       <p>Expedition excursion design darn excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-      <img src="img/destination.jpg" alt="Second slide">
+      <img id="pic3" src="img/destination.jpg" alt="Second slide">
     </section>
     <section class="content-pick">
 
       <div class="destination">
         <h4>Fun In The Sun</h4>
         <p>Expedition excursion design excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-        <div class="btn">Sign Me Up!</div>
+        <a href="#" class="btn" id="btnfun">Sign Me Up!</a>
```

Comment:
Was there a particular reason that you felt you needed to add anchor tags and ids to the buttons? :thinking: